apps/keyboard/js/layouts/my.js

```diff
@@ -5,31 +5,38 @@ Keyboards.my = {
   secondLayout: true,
   basicLayoutKey: '\u1000\u1001\u1002', /*ကခဂ*/
   alternateLayoutKey: '\u1041\u1042\u1043', /*၁၂၃*/
-  imEngine: 'latin',
+  imEngine: 'myanmar',
```

Comment:
You would need add this IME to `apps/keyboard/build/layout_configurator.js` L116 or it would cause build failure.